.htaccess

```
@@ -41,11 +41,12 @@
 
   # Add cache control for static resources
   <FilesMatch "\.(css|js|svg|gif|png|jpg|ico|wasm|tflite)$">
-    Header set Cache-Control "max-age=15778463"
-  </FilesMatch>
-
-  <FilesMatch "\.(css|js|svg|gif|png|jpg|ico|wasm|tflite)(\?v=.*)?$">
-    Header set Cache-Control "max-age=15778463, immutable"
+    <If "%{QUERY_STRING} =~ /(^|&)v=.*/">
```

```suggestion
    <If "%{QUERY_STRING} =~ /(^|&)v=/">
```
Leading and trailing wildcards are generally redundant in regex.

---

Indeed, removed it.